libraries/AP_Math/AP_Math.cpp

```diff
@@ -338,6 +338,22 @@ bool rotation_equal(enum Rotation r1, enum Rotation r2)
     return (v1 - v2).length() < 0.001;
 }
 
+/*
+ * return a velocity correction (in m/s in NED) for a sensor's position given it's position offsets
+ * sensor_offset_bf is in meters in body frame (Foward, Right, Down)
+ * rot_ef_to_bf is a rotation matrix to rotate from earth-frame (NED) to body frame
+ * angular_rate is rad/sec
+ */
```

Comment:
Add another line to the comment to indicate that the correction should added to the sensor NED measurement (not subtracted). It is not obvious from the code unless you can understand how the cross-product operator is working.

Reply:
Thanks, resolved in a follow-up commit.